appcli/orchestrators.py

```diff
@@ -406,6 +445,19 @@ def task(
         command.extend(list(extra_args))
         return self.__compose_task(cli_context, command)
 
+    def exec(
+        self,
+        cli_context: CliContext,
+        service_name: str,
+        command_name: str,
+        extra_args: Iterable[str] = [],
+    ) -> CompletedProcess:
+        command = ["docker", "exec"]  # Command is: exec SERVICE COMMAND [ARGS]
+        command.append(service_name)
+        command.append(command_name)
+        command.extend(list(extra_args))
+        return self.__exec_command(" ".join(command))
```

Comment:
See comment below. `__exec_command` should be taking a list of strings, so don't do a join here

Reply:
`" ".join()` has been removed.